web/src/components/Chip/Chip.scss

```
@@ -8,4 +8,5 @@
   padding: 0.3rem 0.5rem;
   margin: 0.25rem;
   cursor: pointer;
+  @include general.button-hover;
```

```suggestion
  @include general.highlight-hover;
```

Turns out button-hover includes a border-radius: 1rem which doesn't line up with chip's border-radius, so I think we should spit this up so highlight-hover just applies the background-color: $secondary and button-hover includes that and the border-radius. Maybe even rename button-hover to like rounded-button-hover or something.

---

Also it appears that 'cursor: pointer' is always used alongside '@include general.button-hover', so that could be added to the 'general.button-hover' or 'general.highlight-hover' mixins.